tomni/annotation_manager/annotations/polygon/main.py

```diff
@@ -222,16 +310,21 @@ def to_binary_mask(self, shape: Tuple[int, int]) -> np.ndarray:
         return mask
 
     def _calculate_area(self) -> None:
-        self._area = cv2.contourArea(self._contour)
+        # outer area for calculating circularity f.e.
+        self._outer_area = cv2.contourArea(self._contour)
+        self._area = self._outer_area
```

Comment:
just do self._area in 1 go and substract the inner areas instead of renaming a variable

Reply:
or have a temp area and then at the end self._area = temp_variable